### PR TITLE
materialdesign theme

### PR DESCRIPTION
**Pull request by szha**
## Description ##
try materialdesign theme

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)

### CODE REVIEW

**Comment by mli** [comment]
Job PR-383/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/8/index.html

**Comment by mli** [comment]
Job PR-383/14 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/14/index.html

**Comment by mli** [comment]
Job PR-383/15 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/15/index.html

**Comment by mli** [comment]
Job PR-383/22 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/22/index.html

**Comment by mli** [comment]
Job PR-383/24 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/24/index.html

**Comment by mli** [comment]
Job PR-383/33 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/33/index.html

**Comment by mli** [comment]
Job PR-383/37 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/37/index.html

**Comment by mli** [comment]
Job PR-383/43 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/43/index.html

**Comment by mli** [comment]
Job PR-383/44 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/44/index.html

**Comment by mli** [comment]
Job PR-383/58 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/58/index.html

**Comment by mli** [comment]
Job PR-383/61 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/61/index.html

**Comment by mli** [comment]
Job PR-383/62 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/62/index.html

**Comment by mli** [comment]
Job PR-383/64 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/64/index.html

**Comment by eric-haibin-lin** [comment]
Some comments about the layout. If time is limited we can merge this PR first and fix these later: 

screen shot 2018-11-20 at 1 02 52 pm


screen shot 2018-11-20 at 1 03 09 pm

The spacing for the first sentence is a bit weird (add more space?)

The command doesn't render very well. We can probably follow gluon-cv to link the command instead of showing it directly 
screen shot 2018-11-20 at 1 02 13 pm

File: docs/conf.py [hunk 1]
```diff
@@ -183,16 +207,16 @@
 htmlhelp_basename = project + 'doc'
 
 # -- Options for LaTeX output ---------------------------------------------
-latex_elements = {
-}
+# latex_elements = {
```
[the remark this answers is not in the document]
Comment (szha) [reply docs/conf.py]: This is part of the sphinx template and having the commented code makes it easier to add it back.

**Comment by szha** [comment]
@eric-haibin-lin somehow the first header is hidden, and hence the weird layout. I added a dummy/duplicate header as a hack in some pages but I'm yet to solve it systematically.

**Comment by mli** [comment]
Job PR-383/65 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/65/index.html

**Comment by mli** [comment]
Job PR-383/69 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/69/index.html

**Comment by mli** [comment]
Job PR-383/71 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/71/index.html

**Comment by mli** [comment]
Job PR-383/75 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/75/index.html

**Comment by mli** [comment]
Job PR-383/76 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/76/index.html

**Comment by mli** [comment]
Job PR-383/77 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/77/index.html

**Comment by mli** [comment]
Job PR-383/81 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/81/index.html

**Comment by eric-haibin-lin** [comment]
Some of the texts are truncated on mobile:

wechatimg107
wechatimg108
wechatimg109

**Comment by mli** [comment]
Job PR-383/82 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-383/82/index.html

**Comment by szha** [comment]
@eric-haibin-lin currently the theme is shared between the mxnet new-docs and gluonnlp, so this will be addressed in mx-theme for the two projects together.

**Pull request merged**